miriway-session.in

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh -eu
+
+exec env MIRIWAY_SESSION_STARTUP="@CMAKE_INSTALL_FULL_LIBEXECDIR@/miriway-session-startup" MIRIWAY_SESSION_SHUTDOWN="@CMAKE_INSTALL_FULL_LIBEXECDIR@/miriway-session-shutdown" miriway "$@"
```

Comment:
Instead of using `exec`, you might want to use `exec systemd-cat --identifier=miriway` so that logs go to the journal.